src/WebJobs.Script.WebHost/Controllers/FunctionsController.cs

```diff
@@ -37,6 +29,8 @@ public FunctionsController(WebScriptHostManager scriptHostManager, WebHookReceiv
 
         public override async Task<HttpResponseMessage> ExecuteAsync(HttpControllerContext controllerContext, CancellationToken cancellationToken)
         {
+            await _scriptHostManager.DelayUntilHostReady();
```

Comment:
I have to inline this here as opposed to using the new attribute because this isn't an action, and filter attributes only work for actions.